guides/v2.2/cloud/project/project-conf-files_magento-app.md

```diff
@@ -138,6 +138,78 @@ web:
 
 ```
 
+Use the following example of configuration if you deploy Magento in the not a root folder but in a subfolder - for example to `magento_src`:
```

Comment:
> Also, what is the use case for installing a project in a sub-folder? Providing that context will make the example more effective, I think.

@osrecio , could you please share your use case? :)

Thank you!

Reply:
> If we want to provide details about a custom configuration, it would be better to have a separate topic.

yes, this is a custom configuration. Moved to a new topic:

- https://devdocs.magento.com/guides/v2.2/cloud/project/project-deploy-into-subfolder.html
- https://devdocs.magento.com/guides/v2.3/cloud/project/project-deploy-into-subfolder.html

Reply:
Checking with our product team about whether this info should be including in the Cloud Guide since it is not a common use case. Will follow up.

Reply:
ok :)

Thank you!